File: Python Practice/leap_year.py
a = float(input("Enter the year: "))

if a % 4 == 0 and a % 100 != 0:
    print("It is a leap year")

elif a % 400 == 0 and a % 100 == 0:
    print('It is a leap year')

else:
    print("It is not a leap year")

File: Java/leapYear.java
package programs.java;

public class leapYear {
    public static void main(String[] args) {
        int years = 2200;

        if(years%400 == 0){
            System.out.println("Its a leap year");
        }
        else if (years%100 == 0) {
            System.out.println("Ita not a leap year");
        } else if (years%4==0) {
            System.out.println("Its a leap year");
        }
        else {
            System.out.println("Its not a leap year");
        }
    }
}
